instrumentation-api-incubator/build.gradle.kts

```diff
@@ -45,16 +69,8 @@ tasks {
     dependsOn("generateJflex")
   }
 
-  val testStableSemconv by registering(Test::class) {
-    jvmArgs("-Dotel.semconv-stability.opt-in=database,code")
-  }
-
-  val testBothSemconv by registering(Test::class) {
-    jvmArgs("-Dotel.semconv-stability.opt-in=database/dup,code/dup")
-  }
-
   check {
-    dependsOn(testStableSemconv)
-    dependsOn(testBothSemconv)
+    dependsOn(testing.suites.named("testStableSemconv"))
+    dependsOn(testing.suites.named("testBothSemconv"))
```

Comment:
could we have this everywhere for consistency (and so we know we aren't missing any)?

```
check {
  dependsOn(testing.suites)
}
```